test/finance/VestingWallet.test.js

```diff
@@ -62,11 +71,19 @@ describe('VestingWallet', function () {
         this.token = await ethers.deployContract('$ERC20', ['Name', 'Symbol']);
         await this.token.$_mint(this.mock, this.amount);
 
-        this.getBalance = account => this.token.balanceOf(account);
```

Comment:
We don't need this functions anymore now that we're using `changeTokenBalances` and `changeEtherBalances`